packages/edit-site/src/components/header/index.js

```diff
@@ -126,7 +126,6 @@ export default function Header( { openEntitiesSavedStates } ) {
 					ids={ settings.templateIds }
 					templatePartIds={ settings.templatePartIds }
```

Comment:
This can be removed right?

Reply:
The `templatePartIds` can now, yep! I'll push a commit. (The template `ids` are still needed for `add-template` -- I plan to remove those in another PR.)

Reply:
00126c6 and 583b19a